docs/static/ls-ls-native.asciidoc

```diff
@@ -36,15 +36,13 @@ input {
 
 In order to obtain the best performance when sending data from one Logstash to another, the data is batched and compressed. As such, the upstream Logstash (the sending Logstash) only needs to be concerned about configuring the receiving endpoint with these options:
 
-* `hosts` - The receiving Logstash and port. If no port specified, 9800 will be used.
-
-NOTE: In the future, {ls} will support multiple output hosts.
+* `hosts` - The receiving Logstash host and port pairs. If no port specified, 9800 will be used.
```

Comment:
@roaksoax WDYT?

```suggestion
* `hosts` - The receiving one or more Logstash host and port pairs. If no port specified, 9800 will be used.

NOTE: {ls} _fairly_ distributes batched events to _all_ of configured downstream hosts. The fairness is based on, a  downstream that is running, has a capacity to receive events and no recent failures.
```

Reply:
Let's pick one place where we put the explanation. Maybe we should keep it here, and not on the other part.